assets/section-image-banner.css

```diff
@@ -5,25 +5,58 @@
 }
 
 @media screen and (max-width: 749px) {
-  .banner--mobile-bottom:not(.banner--stacked):not(.banner--adapt)
-    > .banner__media {
+  .banner--small.banner--mobile-bottom:not(.banner--adapt) > .banner__media,
```

Comment:
These selectors for defining height may benefit from a refactor to ensure increased code readability and avoiding selector repetition. Something like this:
``` css
.banner--adapt {
  --image-banner-height: initial;
  --image-banner-height-desktop: initial;
}

.banner--small {
  --image-banner-height: 28rem;
  --image-banner-height-desktop: 42rem;
}

.banner--medium {
  --image-banner-height: 34rem;
  --image-banner-height-desktop: 56rem;
}

.banner--large {
  --image-banner-height: 39rem;
  --image-banner-height-desktop: 72rem;
}

@media screen and (max-width: 749px) {
  .banner--mobile-bottom > .banner__media,
  .banner--stacked:not(.banner--mobile-bottom) > .banner__media {
    height: var(--image-banner-height);
  }

  .banner:not(.banner--mobile-bottom) .banner__content {
    min-height: var(--image-banner-height);
  }
}

@media screen and (min-width: 750px) {
  .banner {
    flex-direction: row;
    min-height: var(--image-banner-height-desktop);
  }
}

```